src/app/organizer/ItemTable.tsx

```diff
@@ -179,6 +179,16 @@ function ItemTable({
     [terminal, items]
   );
 
+  const recencyIndicies: Map<string, number> = useMemo(
```

Comment:
I like this concept, but it does make the `columns` value now depend, indirectly, on `items` - meaning we now re-render the entire table every time the items change. I'd love to avoid that if we can.

Reply:
> I like this concept, but it does make the `columns` value now depend, indirectly, on `items` - meaning we now re-render the entire table every time the items change. I'd love to avoid that if we can.

Yeah, I was hesitant to introduce this for that reason. I had noticed Columns was refactored for performance a while ago to avoid just this situation.

Any suggestions on decoupling the two? Wondering if there's elsewhere in the code I can reference as a model for improving the performance on this. I had two ideas when working on it:

1. Calculate the recency index for all items, relative to all other items and store it as an additional property. This could be done when fetching items from the API, will add a small performance hit there, but will be constant after loading. Potential downside is the recency value will be relative to the entire inventory, rather than the currently filtered items.
2. Add a check prior to calculating the recency index to see if the "Recency" column is enabled. 

Any thoughts would be much appreciated, still learning my way around this codebase! 😄

Reply:
Yeah it's just a tricky one in general. Maybe we can just start with an empty column?

Reply:
Yeah I agree, definitely the simplest approach, I'll update this PR sometime tomorrow and attach a new gif of it in action 👍

Reply:
Updated to remove the values, let me know what you think 

![RecencySortNoValues](https://user-images.githubusercontent.com/6667803/121805313-abf80c80-cc42-11eb-90f1-1c77740d175e.gif)